### ISSUE TITLE
Unable to resolve module ./assets/commoncontrolchevronforward.12.svg

### ISSUE THREAD

**Issue by UbiquitousBear** [microsoft/fluentui-react-native#774]
### Environment Information

- **Package version(s)**: `npm 7.0.14`
- **OS version**: `macOS 11.4`

- **Platform**:
  - [ ] iOS
    - **Xcode version**:
  - [x] macOS
    - **Xcode version**:
  - [ ] win32
  - [ ] windows
  - [ ] android

#### Actual behavior:

Received this error when starting the application:

```
error: Error: Unable to resolve module ./assets/commoncontrolchevronforward.12.svg from /Users/shamil/Projects/substratium/node_modules/@fluentui-react-native/contextual-menu/lib-commonjs/SubmenuItem.js: ./assets/commoncontrolchevronforward.12.svg could not be found within the project.

If you are sure the module exists, try these steps:
 1. Clear watchman watches: watchman watch-del-all
 2. Delete node_modules and run yarn install
 3. Reset Metro's cache: yarn start --reset-cache
 4. Remove the cache: rm -rf /tmp/metro-*
  47 | var icon_1 = require("@fluentui-react-native/icon");
  48 | var interactive_hooks_2 = require("@fluentui-react-native/interactive-hooks");
> 49 | var commoncontrolchevronforward_12_svg_1 = __importDefault(require("./assets/commoncontrolchevronforward.12.svg"));
     |                                                                     ^
  50 | exports.SubmenuItem = foundation_compose_1.compose({
  51 |     displayName: SubmenuItem_types_1.submenuItemName,
  52 |     usePrepareProps: function (userProps, useStyling) {
    at ModuleResolver.resolveDependency (/Users/shamil/Projects/substratium/node_modules/metro/src/node-haste/DependencyGraph/ModuleResolution.js:191:15)
    at DependencyGraph.resolveDependency (/Users/shamil/Projects/substratium/node_modules/metro/src/node-haste/DependencyGraph.js:353:43)
    at Object.resolve (/Users/shamil/Projects/substratium/node_modules/metro/src/lib/transformHelpers.js:271:42)
    at resolve (/Users/shamil/Projects/substratium/node_modules/metro/src/DeltaBundler/traverseDependencies.js:571:33)
    at /Users/shamil/Projects/substratium/node_modules/metro/src/DeltaBundler/traverseDependencies.js:587:26
    at Array.reduce (<anonymous>)
    at resolveDependencies (/Users/shamil/Projects/substratium/node_modules/metro/src/DeltaBundler/traverseDependencies.js:586:33)
    at /Users/shamil/Projects/substratium/node_modules/metro/src/DeltaBundler/traverseDependencies.js:275:33
    at Generator.next (<anonymous>)
    at asyncGeneratorStep (/Users/shamil/Projects/substratium/node_modules/metro/src/DeltaBundler/traverseDependencies.js:87:24)
```

from the following code:

```
import React from 'react';
import { Text, View } from 'react-native';
import { PrimaryButton } from '@fluentui/react-native';

const HelloWorldApp = () => {
  return (
    <View
      style={{
        flex: 1,
        justifyContent: "center",
        alignItems: "center"
      }}>
      <Text>Hello, world! :)</Text>
      <PrimaryButton></PrimaryButton>
    </View>
  )
}
export default HelloWorldApp;
```

#### Expected behavior:

Application should build and display the UI without errors.

### Priorities and help requested:

Are you willing to submit a PR to fix? (Yes, No)

Requested priority: (Blocking, High, Normal, Low)

Products/applications affected: (if applicable)

**Comment by Saadnajmi** [comment]
Thanks for reporting this!

**Comment by Saadnajmi** [comment]
Hello, this should be fixed with #782 , if you try pulling in the latest version of the package from nom!
